src/JunctionTreeTypes.jl

```diff
@@ -14,14 +14,17 @@ mutable struct BayesTree
   btid::Int
   cliques::Dict{Int,Graphs.ExVertex}
   frontals::Dict{Symbol,Int}
+  #TEMP JT for evaluation, to store message channels in tree, would be better stored on edges
```

Comment:
thats fine, continue on that path.  I think the intermediate solution will be to use wrapper function `getCliqMsgUp` etc.